test/d-b-n_smoke/arrays.html

```diff
@@ -286,7 +317,8 @@
                             // check and wait for async dom-binds stamping
                             setTimeout(function listener() {
                                 expect(patchAppliedEventCallback).to.have.been.called;
-                                expect(document.querySelector("ul.push>li:nth-child(" + model.people_push.length + ")>span").innerHTML).to.equal("Wizard from Oz");
+                                // nth-child is 1-indexed
```

Comment:
Actually... we already have this one. That's the place we're looking at here. Or maybe I have misunderstood you?